src/borg/archiver.py

```diff
@@ -142,6 +142,13 @@ def wrapper(self, args, repository, key, manifest, **kwargs):
     return wrapper
 
 
+def parse_storage_quota(storage_quota):
+    parsed = parse_file_size(storage_quota)
+    if parsed < parse_file_size('10M'):
+        raise argparse.ArgumentTypeError('quota is too small (%s), must exceed 10M' % storage_quota)
```

Comment:
"exceed" (from: excessive) sounds strange here. rather "must be at least 10M"?

also: refactor to not repeat '10M'?